scripts/fuzz_opt.py

```diff
@@ -1225,7 +1225,7 @@ def randomize_opt_flags():
                 print('avoiding --flatten due to multivalue + reference types not supporting it (spilling of non-nullable tuples)')
                 continue
             if '--gc' not in FEATURE_OPTS:
-                print('avoiding --flatten due to GC not supporting it (spilling of RTTs)')
+                print('avoiding --flatten due to GC not supporting it (spilling of non-nullable locals)')
```

Comment:
I don't think this will be true once we support 1a. Might be worth a TODO here for that.